mne/io/nicolet/nicolet.py

```diff
@@ -0,0 +1,204 @@
+        ECG channels. If None (default), the channel names beginning with
+        ``ECG`` are used.
+    misc : list or tuple
+        Names of channels or list of indices that should be designated
+        MISC channels. If None, (default) none of the channels are designated.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    raw : Instance of Raw
+        A Raw object containing the data.
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+    """
+    return RawNicolet(input_fname, montage=montage, eog=eog, misc=misc,
+                      verbose=verbose)
+
+
+def _get_nicolet_info(fname, eog, ecg, misc):
+    """Function for extracting info from Nicolet header files."""
+    info = _empty_info()
+    fname = fname.split('.')[0]
+    header = fname + '.head'
+    if not path.exists(header):
+        raise IOError('Cannot find header file %s.' % header)
+
+    logger.info('Reading header...')
+    header_info = dict()
+    with open(header, 'r') as fid:
+        for line in fid:
+            var, value = line.split('=')
+            if var == 'elec_names':
+                value = value[1:-2].split(',')  # strip brackets
+            elif var == 'conversion_factor':
+                value = float(value)
+            elif var != 'start_ts':
+                value = int(value)
+            header_info[var] = value
+
+    ch_names = header_info['elec_names']
+    if eog is None:
+        eog = [idx for idx, ch in enumerate(ch_names) if ch.startswith('EOG')]
+    if ecg is None:
+        ecg = [idx for idx, ch in enumerate(ch_names) if ch.startswith('ECG')]
+    if misc is None:
+        misc = list()
+
+    info['filename'] = fname
+
+    date, time = header_info['start_ts'].split()
+    date = date.split('-')
+    time = time.split(':')
+    sec, msec = time[2].split('.')
+    date = datetime.datetime(int(date[0]), int(date[1]), int(date[2]),
+                             int(time[0]), int(time[1]), int(sec), int(msec))
+    info['meas_date'] = calendar.timegm(date.utctimetuple())
+    info['nchan'] = header_info['num_channels']
+    info['sfreq'] = header_info['sample_freq']
+    info['ch_names'] = ch_names
+
+    # Some keys to be consistent with FIF measurement info
+    info['description'] = None
+    info['buffer_size_sec'] = 10.
+
+    cal = header_info['conversion_factor'] * 1e-6
+    for idx, ch_name in enumerate(ch_names):
+        chan_info = {}
+        chan_info['cal'] = cal
+        chan_info['logno'] = idx + 1
+        chan_info['scanno'] = idx + 1
+        chan_info['range'] = 1.0
+        chan_info['unit_mul'] = 0.
+        chan_info['ch_name'] = ch_name
+        chan_info['unit'] = FIFF.FIFF_UNIT_V
+        chan_info['coord_frame'] = FIFF.FIFFV_COORD_HEAD
+        chan_info['coil_type'] = FIFF.FIFFV_COIL_EEG
+        chan_info['kind'] = FIFF.FIFFV_EEG_CH
+        chan_info['loc'] = np.zeros(12)
```

Comment:
why create a dict and then assigne in each line? this goes in one constructor expression, no?